src/components/navbar/navbar.jsx

```diff
@@ -0,0 +1,33 @@
+import "./navbar.css";
+import pic from "../../assets/logo.png";
+
+const Navbar = () => {
+    return (
+
+        <header>
+        <div className="navbar">
+            <div className="nav-items flex-flow-row">
+                <div className="nav-item"><img src={pic} height="40rem" width="100rem"/></div>
```

Comment:
Suggestion: you can extracts inline styling from here and make a class and use them

Reply:
Actually, I think this is different from inline styling. Adding height and width directly to img are kinda attributes.